src/content/resources/javascript/es6/arrow-functions.md

```diff
@@ -1,11 +1,59 @@
 ---
 authors:
   - "veksen#1060"
+  - "t0m#5956"
 created_at: 2019/10/08
-updated_at: 2019/10/08
+updated_at: 2019/10/13
 title: Arrow functions
 ---
+## Arrow Function Syntax
 
-## Arrow functions
+Arrow functions are a new way to declare functions within ES6. Arrow functions allow for a shorter syntax. For example, below is a regular function, as usually seen before ES6.
+```javascript
+const sayHello = function() {
+  console.log('hello');
+}
+```
+The same function can be rewritten with arrow functions, as seen below.
+```javascript
+const sayHello = () => {
+  console.log('hello');
+}
+```
+Both of the above functions have no parameters, but if you want to pass parameters into the function with the arrow syntax, you can do the following, placing the parameters within the brackets of the arrow function.
+```javascript
+const sayHello = (nameOne,nameTwo) => {
+  console.log(nameOne,'says hello to',nameTwo);
+}
+```
+If you want to pass only one parameter, you don't need the brackets around the parameters, and can do the following:
+```javascript
+const sayHello = name => {
+  console.log('hello',name);
+}
+```
+Another interesting feature that also reduces the syntax within arrow functions, is that if a function only has one statement (line) and that line needs to return a value, you can omit the ``return`` statement, and the brackets. Be aware that in multiple line arrow functions, the return keyword still needs to be used. The following function returns "hello":
+```javascript
+const sayHello = () => return "hello";
```

Comment:
Might be worth mentioning that you need the return statement if you open the function in addition to what @Aiden01 mentioned, example

```js
const x = () => {
  return 1
}
```

Reply:
I did, "Be aware that in multiple line arrow functions, the return keyword still needs to be used."

Reply:
Didn't notice your commit right before my review was submitted. You can mark this as resolved